### PR TITLE
[bug] celery burying logs

### PR DESCRIPTION
**Pull request by racheldaniel**
## What is this PR?
This is a:
- [ ] documentation update
- [x] bug fix with no breaking changes
- [ ] new functionality
- [ ] a breaking change

All pull requests from community contributors should target the `main` branch (default).

## Description & motivation
Prior to this change we could not see any logs passing through celery in stdout/k8s. This included log messages containing exception details needed for debugging. 

This PR will move root logger configuration to the celery app, formatting celery logs  according to how we already format logs from uvicorn and our custom dbt-server logger, and will echo everything output into that celery logfile to stdout.

NOTE-- tests will break on this pr until[ this other PR](https://github.com/dbt-labs/dbt-server/pull/213) has been merged

## Checklist
- [x] I have verified that these changes work locally on the following warehouses (Note: it's okay if you do not have access to all warehouses, this helps us understand what has been covered)
    - [ ] BigQuery
    - [ ] Postgres
    - [ ] Redshift
    - [x] Snowflake
    - [ ] Databricks
    - [ ] Spark
- [ ] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models
- [ ] I have added an entry to CHANGELOG.md

### CODE REVIEW

File: bash/ubuntu-run.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ done <<< "$env_vars"
 service redis-server start
 service celeryd start
 
+tail -f /var/log/celery/celery-all.log &
```
Comment (isaac-taylor) [inline bash/ubuntu-run.sh:25]: the magic line :P

File: dbt_server/logging.py [hunk 2]
```diff
@@ -41,42 +41,55 @@ def add_fields(self, log_record, record, message_dict):
...
+formatter = get_log_formatter()
+stdout.setFormatter(formatter)
+DBT_SERVER_LOGGER.addHandler(stdout)
 
 # make sure uvicorn is deferring to the root
 # logger to format logs
 logger_instance = logging.root.manager.loggerDict.get("uvicorn")
 if logger_instance:
-    logger.propagate = True
+    logger_instance.propagate = True
```
Comment (isaac-taylor) [inline dbt_server/logging.py:73]: ah interesting find. Is this a separate bug?

File: dbt_server/logging.py [hunk 3]
```diff
@@ -41,42 +41,55 @@ def add_fields(self, log_record, record, message_dict):
...
 
 
+def get_configured_celery_logger():
+    logger = logging.getLogger(__name__)
+    logger.handlers = []
+    formatter = get_log_formatter()
+    file_handler = logging.FileHandler(filename=CELERY_LOG_FILE.get())
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+    logger.setLevel(logging.INFO)
```
Comment (isaac-taylor) [inline dbt_server/logging.py:88]: does this level get propagates down to dbt somehow?

File: dbt_worker/tasks.py [hunk 4]
```diff
@@ -107,19 +108,18 @@ def _invoke_runner(
         # artifacts may write to incorrect locations
         if project_dir:
             os.chdir(project_dir)
-        # TODO: Make sure callback works
-        dbt = dbtRunner(callbacks=[log_event_to_console])
+        dbt = dbtRunner()
         dbt.invoke(command)
+        logger.info(f"Task with id: {task_id} has completed successfully")
```
Comment (isaac-taylor) [inline dbt_worker/tasks.py:113]: Maybe just "completed"?

"successfully" might confuse us later and think the dbtRunner itself was successful?

**Review by isaac-taylor — APPROVED** [review]
Looks reasonable to me!

Minor comments/questions, mostly for my own education

File: dbt_server/logging.py (hunk 2 above)
Comment (racheldaniel) [reply dbt_server/logging.py:73]: Yep. It was never doing anything lol. I'd rather go ahead and loop those logs in and then remove them if we decide they're too noise

File: dbt_server/logging.py (hunk 3 above)
Comment (racheldaniel) [reply dbt_server/logging.py:88]: No, this level should not affect the dbt level. We should still be getting debug logs from dbt

**Pull request merged**